pkg/metrics/exporter/export.go

```diff
@@ -26,6 +29,27 @@ type Exporter struct {
 	outputFile string
 }
 
+type MutexExporter struct {
+	mu  sync.Mutex
+	exp *Exporter
+}
+
+func (mexp *MutexExporter) Get() *Exporter {
+	mexp.mu.Lock()
```

Comment:
Why need a lock here? Is there a race condition?

Reply:
There is not a race now. I already removed it.